test/integration/L2OpUSDCBridgeAdapter.t.sol

```diff
@@ -104,12 +104,13 @@ contract Integration_Bridging is IntegrationBase {
     vm.prank(address(l2Adapter));
     bridgedUSDC.mint(_user, _amount);
 
+    uint256 _userBalanceBefore = bridgedUSDC.balanceOf(_user);
```

Comment:
Small change on an integration test to make it more future proof in case something on the setup changes